src/Core/Services/Implementations/LaunchDarklyFeatureService.cs

```diff
@@ -4,16 +4,25 @@
 using LaunchDarkly.Logging;
 using LaunchDarkly.Sdk.Server;
 using LaunchDarkly.Sdk.Server.Integrations;
+using LaunchDarkly.Sdk.Server.Interfaces;
 
 namespace Bit.Core.Services;
 
-public class LaunchDarklyFeatureService : IFeatureService, IDisposable
```

Comment:
ℹ️ This concerns me as it's against guidance from LaunchDarkly, but it's not available when using `ILdClient`. Any alternatives?